ansible/roles/software/ckan/inventory/templates/etc/ckan/production.ini

```diff
@@ -108,8 +108,9 @@ ckan.redis.url = {{ inventory_ckan_redis_url}}
 #       Add ``pdf_preview`` to enable the resource preview for PDFs
 #		Add ``resource_proxy`` to enable resorce proxying and get around the
 #		same origin policy
-# ckan.plugins = usmetadata datajson datastore datapusher stats text_view recline_view googleanalyticsbasic saml2 dcat_usmetadata
-ckan.plugins = {{ (inventory_ckan_plugins_default + inventory_ckan_plugins_additional) | join(' ') }}
+# If saml2auth and usmetadata both present, saml2auth needs to appear before usmetadata, 
+# so the login behaviour in its template overrides usmetadata's. 
+ckan.plugins = {{ (inventory_ckan_plugins_additional + inventory_ckan_plugins_default) | join(' ') }}
```

Comment:
I wouldn't change the order of the variables unless you're also going to change it for catalog. Best to be consistent here.

If you need to control the order, maybe just overwrite the variables:
```yaml
inventory_ckan_plugins_default: []
inventory_ckan_plugins_additional:
  - saml2auth
  - usmetadata
  - ...
```

Not sure if there's a better scheme. I guess we could introduce a new variable inventory_ckan_plugins_preload and then
```
ckan.plugins = {{ (inventory_ckan_plugins_preload + inventory_ckan_plugins_default + inventory_ckan_plugins_additional) | join(' ') }}
```

That would work for catalog too.